FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -44,8 +44,12 @@ class IssueDetailViewController : UIViewController, IssueShareable {
             tableView.deselectRow(at: indexPath, animated: true)
         }
         logs = ContactLogs.load()
-        
-        navigationController?.setNavigationBarHidden(false, animated: true)
+
+        if let nav = self.navigationController, nav.viewControllers.count > 1 {
+            navigationController?.setNavigationBarHidden(false, animated: true)
+        } else {
+            tableView.contentInset = UIEdgeInsets(top: 64, left: 0, bottom: 0, right: 0)
```

Comment:
probably, but I think that would just trade one dependency (a hard coded number) with another (a way to get that value up-and-down the view controller hierarchy). I'm still not sure this design is the approach we want to take for the iPad version. or do we just keep the header in the left-hand controller.